### ISSUE TITLE
"origin not allowed" with InfluxDB "Mixed" datasources, v3.8.6

### ISSUE THREAD

**Issue by andreaaizza** [grafana/grafana#45253]
**What happened**: "origin not allowed" all over the places also with v3.8.6, **when using a `Mixed` InfluxDB datasource**.
Possibly duplicate of some other #45117, #45115

**Environment**:
- Grafana version: v8.3.6
- Data source type & version: InfluxDB
- OS Grafana is installed on: Linux
- User OS & Browser: Ubuntu

**Comment by zuchka** [comment]
Hi @andreaaizza thanks for making this issue. 
Are you running grafana behind a reverse proxy? If so, which one?

**Comment by andreaaizza** [comment]
Yes. Sorry. 'priloxy_pass' by nginx.
Cheers,
Andrea

**Comment by zuchka** [comment]
@andreaaizza 

can you please try editing your NGINX config to include `proxy_set_header Host $http_host;` in your location block?

See this comment, which seems to be helping other users get unblocked here:

https://github.com/grafana/grafana/issues/45117#issuecomment-1033842787
